packages/core/logger/src/index.js

```diff
@@ -197,13 +197,13 @@ export class Logger {
     this.writeToFile.push({ args, origin, prefix: "[trace]" });
   }
 
-  dir(obj) {
-    if (!obj || !(this.shouldLog('info'))) {
+  dir(...args) {
+    if (!args || !(this.shouldLog('info'))) {
```

Comment:
If we're going with `...args` in the signature of `logger.dir` then the test should check `!args.length` instead of `!args` because even in the case of no arguments `args` will be `[]`, which is a truthy value.

But seem my [comment/question re: the change in signature]().

```suggestion
    if (!args.length || !(this.shouldLog('info'))) {
```